builtin/providers/aws/resource_aws_default_network_acl_test.go

```diff
@@ -0,0 +1,435 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+var defaultEgressAcl = &ec2.NetworkAclEntry{
+	CidrBlock:  aws.String("0.0.0.0/0"),
+	Egress:     aws.Bool(true),
+	Protocol:   aws.String("-1"),
+	RuleAction: aws.String("allow"),
+	RuleNumber: aws.Int64(100),
+}
+var defaultIngressAcl = &ec2.NetworkAclEntry{
+	CidrBlock:  aws.String("0.0.0.0/0"),
+	Egress:     aws.Bool(false),
+	Protocol:   aws.String("-1"),
+	RuleAction: aws.String("allow"),
+	RuleNumber: aws.Int64(100),
+}
+
+func TestAccAWSDefaultNetworkAcl_basic(t *testing.T) {
+	var networkAcl ec2.NetworkAcl
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSDefaultNetworkAclDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccAWSDefaultNetworkConfig_basic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccGetWSDefaultNetworkAcl("aws_default_network_acl.default", &networkAcl),
+					testAccCheckAWSDefaultACLAttributes(&networkAcl, []*ec2.NetworkAclEntry{}, 0),
+				),
+			},
+			// Add default ACL rules and veryify plan is empty
```

Comment:
That's a remnant of our previous plan of optionally revoking the rules on initial import. Cleaning up